hardware_interface/rosdoc.yaml

```diff
@@ -1,5 +1,5 @@
 - builder: doxygen
   name: C++ API
   output_dir: c++
-  file_patterns: '*.c *.cpp *.h *.cc *.hh *.dox'
+  file_patterns: '*.c *.cpp *.h *.cc *.hh *.dox *.md'
```

Comment:
This is required to use a mardown file (`README.md`) as doxygen's mainpage. See also [`rosdoc_lite`](http://wiki.ros.org/Doxygen#Embedding_Markdown_Pages) how to include markdown files. Only "problem" is that the `\htmlinclude manifest.html` is missing, which is usually present in default `mainpage.dox` of ROS packages.